Src/Support/Google.Apis.Auth.PlatformServices/Google.Apis.Auth.PlatformServices.csproj

```diff
@@ -5,6 +5,7 @@
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <AssemblyOriginatorKeyFile>..\..\..\google.apis.snk</AssemblyOriginatorKeyFile>
+    <IsPackable>false</IsPackable>
```

Comment:
This is because it's packed within Google.Apis.Auth with the weird bit of msbuild, yes?